assets/ui.css

```diff
@@ -401,3 +401,18 @@
     display: none;
   }
 }
+
+#xref-ui {
+  width: 100%;
+  min-height: 550px;
+  height: 100%;
+  overflow: hidden;
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+#xref-ui:not(.ready) {
+  background: url("http://loadinggif.com/images/image-selection/8.gif")
```

Comment:
Where can we host it?

Reply:
can we inline it?

Reply:
(also, is there an SVG version?)

Reply:
Inline: Can do depending on size. This gif is 3KB.
I liked this gif 😺 Will see if I can find a similar SVG

Reply:
The gif is nice. We can just host it on ReSpec.org?